cpu/cc26x0/include/cc26x0_rfc.h

```diff
@@ -160,8 +108,6 @@ typedef struct {
 #define RFC_DBELL_CMDSTA_RESULT_QUEUEERROR      0x88
 #define RFC_DBELL_CMDSTA_RESULT_QUEUEBUSY       0x89
 
-
-
```

Comment:
pew, i've got admit i didn't think anybody was going to look at this in this state...
so, i guess i don't get the opportunity anymore :-/

but, could you change whitespace in separate commits?